docs/book/src/topics/ipv6.md

```diff
@@ -0,0 +1,135 @@
+# IPv6 clusters
+
+## Overview
+
+CAPZ enables you to create IPv6 Kubernetes clusters on Microsoft Azure.
+
+- IPv6 support is available for Kubernetes version 1.18.0 and later on Azure.
+- IPv6 support is in beta as of Kubernetes version 1.18 in Kubernetes community.
+
+To deploy a cluster using IPv6, use the [ipv6 flavor template](https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/master/templates/cluster-template-ipv6.yaml).
+
+:warning:  Action required: The Azure DNS nameserver is only IPv4. If the coredns pod runs on the pod network, it will fail to resolve. 
```

Comment:
nothing major but doesn't look like the emojis are working in the rendered doc.

Reply:
sad, they work in markdown on github... will look into it, otherwise I'll just change it to `WARNING`

Reply:
check this out https://deploy-preview-646--kubernetes-sigs-cluster-api-provider-azure.netlify.app/topics/ipv6.html  😃